portal/views/clinical.py

```diff
@@ -207,6 +210,9 @@ def pca_diag_set(patient_id):
             value:
               type: boolean
               description: the patient's PCa diagnosis
+            status:
```

Comment:
might be worth making a schema for this to enumerate all the values we accept, but it's a pain to get just right